internal/arc/arc_query_tx.go

```diff
@@ -0,0 +1,86 @@
+package arc
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/bitcoin-sv/go-broadcast-client/config"
+	"github.com/bitcoin-sv/go-broadcast-client/internal/decoder"
+	"github.com/bitcoin-sv/go-broadcast-client/internal/httpclient"
+	"github.com/bitcoin-sv/go-broadcast-client/models"
+	"github.com/bitcoin-sv/go-broadcast-client/shared"
+)
+
+var ErrMissingHash = fmt.Errorf("missing tx hash")
```

Comment:
Why not `errors.New(...)`

Reply:
You are right - had some import problems before. Fixed.